about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/stylesheets/css/all.css">
+    <link rel="stylesheet" href="/stylesheets/style.css">
+    <title>About Me</title>
+</head>
+<body>
+    <!-- main -->
+    <main id="about_main">
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="./index.html">Home</a></li>
+                    <li><a href="./portfolio.html">Portfolio</a></li>
+                    <li><a href="./hobby-blog.html">Blog</a></li>
```

Comment:
These links don't work.

```suggestion
                    <li><a href="index.html">Home</a></li>
                    <li><a href="portfolio.html">Portfolio</a></li>
                    <li><a href="hobby-blog.html">Blog</a></li>
```